letsencrypt/client/client.py

```diff
@@ -342,7 +342,8 @@ def init_key(key_size):
         key_pem = crypto_util.make_key(key_size)
     except ValueError as err:
         logging.fatal(str(err))
-        logging.info("Note: The default RSA key size is %d bits.", CONFIG.RSA_KEY_SIZE)
+        logging.info("Note: The default RSA key size is %d bits.",
+                     CONFIG.RSA_KEY_SIZE)
```

Comment:
also, i find the 80chars suggestion of pep8 antiquated (similar like that ^L suggestion for paginating your source) and rather harmful if it leads to stuff like seen above.

I use 120 in my projects. :) YMMV.